lib/puppet/type/yumrepo.rb

```diff
@@ -259,17 +259,11 @@
   end
 
   newproperty(:priority) do
-    desc "Priority of this repository from 1-99. Requires that
+    desc "Priority of this repository. Requires that
```

Comment:
It would be good to leave some documentation about what exactly the value looks like. Something like "Priority of this repository. Can be any integer value (including negative)."?

I can update this when merging in the PR if you don't think you'll have time to update the documentation string.